clang/include/clang/CodeGen/ModuleBuilder.h

```diff
@@ -28,7 +29,7 @@ namespace llvm {
 }
 
 // Prefix of the name of the artificial inline frame.
-#define CLANG_TRAP_PREFIX "__clang_trap_msg"
+inline constexpr llvm::StringRef CLANG_TRAP_PREFIX = "__clang_trap_msg";
```

Comment:
The name should be changed to use the usual variable/constant naming convention (ClangTrapPrefix, I think, in this case).